docs/calicoctl/ipam.md

```diff
@@ -0,0 +1,86 @@
+
+# User guide for 'calicoctl ipam' commands
+
+This section describes the `calicoctl ipam` commands.
+
+This command allows an interface into Calico's IP address management to release 
+IP addresses from endpoints and view additional information about assigned IPs.
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl ipam' commands
+
+Run `calicoctl ipam --help` to display the following help menu for the 
+calicoctl ipam commands.
+
+```
+
+Usage:
+  calicoctl ipam release <IP>
+  calicoctl ipam info <IP>
+
+Description:
+  Manage Calico assigned IP addresses
+
+Warnings:
+  -  Releasing an in-use IP address can result in it being assigned to multiple
+     workloads.
+
+```
+
+## calicoctl ipam commands
+
+
+### calicoctl ipam release <IP>
+
+This command allows you to release an IP address that has been assigned to an 
+endpoint.  When an IP address is released, it becomes available for assignment 
+to any endpoint.
+
+This command can be run on any Calico node. 
+
+Command syntax:
+
+```
+calicoctl ipam release <IP>
+
+    <IP>: IP address to release.
+```
+
+Examples:
+
+```
+$ calicoctl ipam release 192.168.1.1
+Address successfully released
+
+$ calicoctl ipam release fd80:24e2:f998:72d6::1
+Address successfully released
+```
+
+### calicoctl ipam info <IP>
+
+This command prints information about a given IP address, such as special 
+attributes defined for the IP or whether the IP has yet to be assigned to an 
+endpoint.
```

Comment:
I don't believe this is true.  I would rephrase "or whether the IP has been reserved by a user of the Calico IPAM."

We don't check whether it's is in use on an existing endpoint.